bins/stackablectl/src/cli.rs

```diff
@@ -44,7 +46,7 @@ Cached files are saved at '$XDG_CACHE_HOME/stackablectl', which is usually
     pub offline: bool,
 
     /// Namespace in the cluster used to deploy the products and operators
-    #[arg(short, long, default_value = DEFAULT_STACKABLE_NAMESPACE)]
+    #[arg(short, long, default_value = DEFAULT_NAMESPACE)]
     pub namespace: String,
```

Comment:
Thinking of https://github.com/stackabletech/stackablectl/issues/259 I would prefer calling the option `--operators-namespace` with the default value of `DEFAULT_OPERATORS_NAMESPACE`, being `stackable-operators` (or `default` for now).
Going the final way with `stackable-operators` and creating the namespace on demand would be awesome!

Reply:
Works for me as well. I was just thinking "well there are multiple operators running in this namespace".
TLDR: Do whatever you desire :)

Reply:
Changed in 2bbd6f6.

Reply:
I agree with @nightkr, `operator-namespace` flows indeed a little better. Updated in 761a13e.